packages/url_launcher/example/lib/main.dart

```diff
@@ -51,6 +51,15 @@ class _MyHomePageState extends State<MyHomePage> {
     }
   }
 
+  Future<Null> _launchInWebViewWithJavaScript(String url) async {
+    if (await canLaunch(url)) {
+      await launch(url,
+          forceSafariVC: true, forceWebView: true, enableJavaScript: true);
```

Comment:
Nit: Either put all args on one line or use trailing comma (put a comma after the last argument).